File: store-front-end/src/store/modules/goods.js
/*
 * @Description:
 * @Author: chuyang
 * @Date: 2022-06-05 19:48:57
 * @LastEditors: chuyang
 * @LastEditTime: 2022-07-06 16:52:27
 * @FilePath: \store-front-end\src\store\modules\goods.js
 */
const state = {
  id: 0,
  name: "",
  image: "",
  info: "",
  price: "",
};

const getters = {};

const mutations = {
  setGoods(state, payload) {
    state.id = payload.id;
    state.name = payload.name;
    state.image = payload.image;
    state.info = payload.info;
    state.price = payload.price;
  },
};

const actions = {
  updateGoods(content, payload) {
    content.commit("setGoods", payload);
  },
};

export default {
  namespaced: true,
  state,
  getters,
  actions,
  mutations,
};

File: store-back-end/src/main/java/com/chuyang/store/model/Goods.java
package com.chuyang.store.model;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @ClassName Goods
 * @Description TODO:
 * @Author chuyang
 * @Date 2022/7/6 11:23
 * @Version 1.0
 **/
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Goods {
    @TableId(value = "id",type = IdType.AUTO)
    private Integer id;
    private String name;
    private Integer price;
    private String image;
    private String Info;
}
